src/pages/Admin/SignUp/SignUp.js

```diff
@@ -117,43 +170,64 @@ function SignUp({ openSnackbar }) {
   if (success) return <SuccessfulSignUp />;
 
   return (
-    <Layout bg={SignUpBg}>
+    <Layout bg={SignUpBg} style={{ justifyContent: "space-between" }}>
       <TitleComponent title="Novo registo" pageId="signup" />
-      <HeadlineContainer>
-        <Typography variant="h3">{t("admin#signup_title")}</Typography>
-      </HeadlineContainer>
-
-      <Form onSubmit={handleSubmit}>
-        <TextField
-          label={t("admin#signup_nameLabel")}
-          name="defaultQueueName"
-          onChange={e => handleChange(e)}
-          helperText="ex: PDoceCartaxo"
-          inputProps={{ maxLength: 30 }}
-          {...inputProps}
-        />
-
-        <TextField
-          label={t("admin#signup_emailLabel")}
-          name="email"
-          onChange={e => handleChange(e)}
-          {...inputProps}
-        />
-
-        <PasswordInput
-          label={t("admin#signup_passwordLabel")}
-          name="password"
-          onChange={e => handleChange(e)}
-          min="6"
-          {...inputProps}
-        />
-
-        <ButtonsContainer>
-          <Button type="submit" forward style={{ backgroundColor: "unset" }}>
-            {t("admin#signup_register")}
-          </Button>
-        </ButtonsContainer>
-      </Form>
+
+      <MainContainer>
+        <FormContainer>
+          <HeadlineContainer>
+            <Typography variant="h3">{t("admin#signup_title")}</Typography>
+          </HeadlineContainer>
+
+          <Form onSubmit={handleSubmit}>
+            <TextField
+              label={t("admin#signup_nameLabel")}
+              name="defaultQueueName"
+              onChange={e => handleChange(e)}
```

Comment:
this can be simplified to:
```suggestion
              onChange={handleChange}
```